duckduckgo/Android
# Remove returning users experiment.

Task/Issue URL: https://app.asana.com/0/0/1201894743888865/f

### Description
The experiment result was inconclusive, removing the code added with https://github.com/duckduckgo/Android/pull/1541 and https://github.com/duckduckgo/Android/pull/1685 .

### Steps to test this PR

1/  New installs
- install from this branch
- open the app and notice the onboarding
- while on first screen rotate the device to make sure it looks ok

2/ Update from a version that contains the active experiment - Control
- install from tag 5.113.0
- on first screen choose "Let's do it" (Control) and see the first message in onboarding
- update from this branch
- notice you see the onboarding

3/ Update from a version that contains the active experiment - Variant
- install from tag 5.113.0
- clear the storage if you are not in one of the variants
- choose either "Continue Without Privacy Tips" or "Skip Tutorial" depending on the variant
- notice there is no onboarding
- update from this branch
- notice there is no onboarding

### UI changes - normal UI with "Let's Do It" button
